supabase/migrations/20240619042412_rll_policies_org_remove.sql

```diff
@@ -0,0 +1,33 @@
+-- Drop the specific policies for the 'organization' table
+DROP POLICY IF EXISTS "Enable delete for owner" ON public.organization;
+DROP POLICY IF EXISTS "Enable only owner can update" ON public.organization;
+DROP POLICY IF EXISTS "Enable read access for all users" ON public.organization;
+
+-- Drop all existing policies for the 'organization' table if there are any left
+DROP POLICY IF EXISTS delete_policy ON public.organization;
+DROP POLICY IF EXISTS update_policy ON public.organization;
+DROP POLICY IF EXISTS select_policy ON public.organization;
+
+-- Drop the policy Enable read access for all users on 'organization_member' table
+DROP POLICY IF EXISTS "Enable read access for all users" ON public.organization_member;
+
+-- Disable Row-Level Security on the 'organization' table
+ALTER TABLE public.organization DISABLE ROW LEVEL SECURITY;
```

Comment:
why do we drop and then re-add later? more just curious.

Reply:
@chitalian I thought we wanted to keep the RLS but remove the policies, so if we remove the RLS then technically anyone can read and write to org table

Reply:
@chitalian ok this actually doesn't make sense lol